### PR TITLE
Add countries in NIH list which are missing from ENA list

### PR DESCRIPTION
**Pull request by anna-parker**
resolves https://github.com/loculus-project/loculus/issues/2636

preview URL: https://add-nih-countries.loculus.org/

### Summary

As calculated in https://github.com/loculus-project/loculus/pull/2335 these are the countries on the NIH list yet missing from the ENA list, adding them here for consistency. 

For reference here is:
- The ENA checklist: https://www.ebi.ac.uk/ena/browser/api/xml/ERC000011
- The NIH checklist https://www.ncbi.nlm.nih.gov/genbank/collab/country/.

I added these countries to the end of the list. Currently we do not use the ENA checklist for verification - if this changes these countries will need to be properly mapped.

### PR Checklist

- [ ] Check if there are further countries in https://www.insdc.org/submitting-standards/geo_loc_name-qualifier-vocabulary/ that we should add
- [ ] All necessary documentation has been adapted.
- [ ] The implemented feature is covered by an appropriate test.

### CODE REVIEW

File: kubernetes/loculus/values.yaml [hunk 1]
```diff
@@ -395,6 +395,15 @@ defaultOrganismConfig: &defaultOrganismConfig
               - Yemen
               - Zambia
               - Zimbabwe
+              - Eswatini # Add countries in NIH list which are missing from ENA list
```
Comment (corneliusroemer) [inline kubernetes/loculus/values.yaml:398]: I'd probably add them alphabetically?

**Review by corneliusroemer — APPROVED** [review]

**Comment by anna-parker** [comment]
We should probably also add the historical names in case someone submits older data: 
```
Belgian Congo
British Guiana
Burma
Czechoslovakia
Czech Republic
East Timor
Korea
Macedonia
Micronesia
Netherlands Antilles
Serbia and Montenegro
Siam
Swaziland
The former Yugoslav Republic of Macedonia
USSR
Yugoslavia
Zaire
```

**Comment by corneliusroemer** [comment]
How is Czech Republic a historic name? 👀

**Comment by corneliusroemer** [comment]
They now call it Czechia. I see. It would be great if there was a cutoff date for each historical name. One shouldn't use Soviet Union past 1990 etc. We could come up with our own cut offs. Czechia/Czech Republic is weird though.

What about Turkey/Türkiye? It still says Turkey but I bet Turkey would prefer Türkiye.

File: kubernetes/loculus/values.yaml [hunk 2]
```diff
@@ -1557,9 +1571,9 @@ resources:
...
       memory: "3Gi"
 defaultResources:
-    requests:
-      memory: "200Mi"
-      cpu: "20m"
-    limits:
-       memory: "1Gi"
-       cpu: "20m"
\ No newline at end of file
+  requests:
```
Comment (corneliusroemer) [inline kubernetes/loculus/values.yaml:1574]: thanks that's correct indentation now

**Review by corneliusroemer — APPROVED** [review]

File: kubernetes/loculus/values.yaml (hunk 1 above)
Comment (anna-parker) [reply kubernetes/loculus/values.yaml]: done :-)

**Pull request merged**